_profiles/drupal/resource-versioning/index.md

```diff
@@ -0,0 +1,301 @@
+---
+name: Resource Versioning
+short_description: |
+  Defines a protocol for requesting versioned resources via a JSON:API
+  server.
+
+extended_description: |
+  # Overview
+  JSON:API servers that are capable of tracking a record of changes to a
+  resource object may afford the capability of requesting a resource object
+  as it existed in a prior or successive state. This capability is herein
+  defined as _resource versioning_.
+  
+  This profile establishes a protocol for resource versioning by defining a
+  query parameter and its semantics in order to identify arbitrary revisions
+  of a resource and how a server should interpret this query parameter.
+
+minimum_jsonapi_version: 1.0
+minimum_jsonapi_version_explanation:
+
+discussion_url: https://www.drupal.org/project/issues/jsonapi
+
+editors:
+  - name: Gabriel Sullice
+    email: gabriel@sullice.com
+  - name: Mateu Aguiló Bosch
+    email: mateu@mateuaguilo.com
+    website: https://mateuaguilo.com
+  - name: Wim Leers
+    email: work@wimleers.com
+    website: https://wimleers.com
+
+categories:
+  - Resource Versioning
+---
+  
+# Concepts
+Resources on a server may undergo changes and the state of a resource with an
+arbitrary number of changes may be accessible. It is often useful to retrieve
+resources as they existed at the time of their creation or in various states of
+change for editorial or archival purposes. This profile establishes a protocol
+for accessing resources in those various states.
```

Comment:
This is talking about _previous_ states, for archival/history purposes. I think this should also explicitly state _next_ states, i.e. drafts that are actively being worked on, or perhaps need to go through legal review, or are ready but simply aren't yet live because they're _scheduled_ to be published in the future.

Reply:
> This is talking about previous states, for archival/history purposes.

I agree that this could be clarified, but I did not mean previous as you are interpreting it.

Even future revisions were created _previously to now_. They may not be the default version _yet_, but that does not mean that the revision itself was created in the future.